### PR TITLE
 Added Action to change color once start date is reached as requested in Issue #3666

### PR DESCRIPTION
**Pull request by cwolter**
Before to submit your pull-request:

- Be sure that the unit tests pass
- If you create a new feature, test your code, and do not introduce new bugs
- Avoid code duplication
- Remember, small pull-requests are easier to review and can be merged quickly
- 1 pull-request == 1 feature/improvement
- Nonessential features should be implemented as plugin

### CODE REVIEW

File: app/ServiceProvider/ActionProvider.php [hunk 1]
```diff
@@ -98,8 +100,7 @@ public function register(Container $container)
         $container['actionManager']->register(new TaskAssignColorSwimlane($container));
         $container['actionManager']->register(new TaskAssignPrioritySwimlane($container));
         $container['actionManager']->register(new TaskAssignColorOnDueDate($container));
-        $container['actionManager']->register(new SubtaskTimerMoveTaskColumn($container));
-        $container['actionManager']->register(new StopSubtaskTimerMoveTaskColumn($container));
```
Comment (fguillot) [inline app/ServiceProvider/ActionProvider.php]: Why did you remove these actions?

Comment (cwolter) [reply app/ServiceProvider/ActionProvider.php]: Hi, 

This was an error, I fixed it, sorry
